lib/internal/quic/core.js

```diff
@@ -2451,7 +2470,7 @@ class QuicClientSession extends QuicSession {
         default:
           reason = `${handle}`;
       }
-      this.destroy(new ERR_QUICCLIENTSESSION_FAILED(reason));
+      this.destroy(new ERR_OPERATION_FAILED(reason));
```

Comment:
I think it would make sense to map the error codes from the binding to public errors 1:1 here – otherwise we just end up back in a situation where people parse the error message to figure out what is going on

Reply:
Ok, we can definitely do that. I'll separate those out in a commit in the next PR